scripts/restore_backup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+
+############ VARIABLE DEFINITIONS ############
+MEDIA_DIR=/home/dev/BP2021RH1/projects/bp-strapi/public/uploads
```

Comment:
After the pm2 introduction this will lie in `/home/github/...`, no?

Reply:
```suggestion
MEDIA_DIR=/home/github/BP2021RH1/source/projects/bp-strapi/public/uploads
```